File: codeforces/236a.py
s = input()
chars = {}
for x in range(len(s)):
	if x not in chars:
		chars[x] = 1
count = len(chars)
print(count)
if(count % 2 == 0):
	print("CHAT WITH HER!")
else:
	print("IGNORE HIM!")
	

File: codeforces/236a.cpp
//236a.cpp
#include<bits/stdc++.h>

using namespace std;

int main() {
	
	string x;
	map<char,int> chars;
	cin >> x;
	
	unordered_map<char, int> m;
	
	for (int i=0; i<x.length(); i++) {
		m[x[i]]++;
	}
	
	if(m.size() & 1) {
		//odd
		cout << "IGNORE HIM!";
	}
	else {
		cout << "CHAT WITH HER!";
	}
    
	return 0;	
}
